### DISCUSSION TITLE
Server add to list & role badge

### DISCUSSION THREAD

**Discussion by FlexKleks** [mastodon/mastodon#22672] [Q&A]
Hey,
I don't know if this post has been discussed yet. But I have 2 questions about own Mastodon servers.

The first question is how to add your server to the list. So on (https://joinmastodon.org/servers). 

The second question is how to add a "badge" to your account. I once saw an account had a red badge under the name with "Admin". 

Looking forward to answers :-)

**Comment by alansill** [comment]
> Hey,
> 
> I don't know if this post has been discussed yet. But I have 2 questions about own Mastodon servers.
> 
> 
> 
> The first question is how to add your server to the list. So on (https://joinmastodon.org/servers). 
> 
> 
> 
> The second question is how to add a "badge" to your account. I once saw an account had a red badge under the name with "Admin". 
> 
> 
> 
> Looking forward to answers :-)

I have the same question. We filled out the form to get our server added, and provided all the information, a few weeks ago, but at the moment I donʻt even see the link to add one. Is this being overwhelmed?

Thanks in advance for helping build the Fediverse! Having a reliable service to let people find servers would certainly help take the load off of the more popular servers.

**Comment by justdaj** [comment]
I have given up trying to get my two instances listed on JoinMastodon.  I am not sure who curates it, but half the servers are closed to new users.  The annoying thing is, most newbies come to Joinmastodon.org first and are only seeing a tiny handful of the thousands available.

Regarding the badges.  It is a bug in 4.02, which is apparently fixed in the next release.

**Reply by FlexKleks** [reply]
Ohh boy. Bad news. But thanks for helping. Would be nice if Mastodon would offer an application process and an associated review for server listening
